app/services/get_application_choices_for_provider.rb

```diff
@@ -4,10 +4,10 @@ class GetApplicationChoicesForProvider
   def self.call(provider:)
     raise MissingProvider unless provider.present?
 
-    ApplicationChoice.includes(:course)
+    ApplicationChoice.includes(:course, :application_form, :provider, :site, application_form: [:candidate, :references, :application_work_experiences, :application_volunteering_experiences])
       .where('courses.provider_id' => provider)
-      .or(ApplicationChoice.includes(:course)
-      .where('courses.accrediting_provider_id' => provider))
+      .or(ApplicationChoice.includes(:course, :application_form, :provider, :site, application_form: [:candidate, :references, :application_work_experiences, :application_volunteering_experiences])
```

Comment:
😮

Reply:
ikr! Pretty sure this isn't useful in all contexts either...